### ISSUE TITLE
Does not strip fbclid from outgoing Facebook links... 

### ISSUE THREAD

**Issue by nikwolfe7** [jparise/chrome-utm-stripper#35]
I shared the chrome store link for this plugin on Facebook to encourage others to use it. Here's the chrome store link: 

https://chrome.google.com/webstore/detail/tracking-token-stripper/kcpnkledgcbobhkgimpbmejgockkplob

The link which was created when I test-clicked it from Facebook was as follows:

https://chrome.google.com/webstore/detail/tracking-token-stripper/kcpnkledgcbobhkgimpbmejgockkplob?fbclid=IwAR3f0mgjmJz-um_nIKjihvFKNteQmoMO1y1jYeboLWhf9hkFxAHT6tRGYeY

Pretty embarrassing, no? Pls fix.

**Comment by doggy8088** [comment]
It looks like you can't use any extension on Chrome Online Store.

**Comment by toughdiamond** [comment]
Doesn't remove ?fbclid strings for me either. Firefox 68.8.0esr (32-bit).

**Comment by jparise** [comment]
@toughdiamond I'm afraid I can't reproduce this with Firefox 79.0. When I try the example link above, it _is_ stripped correctly using this extension.

**Comment by jparise** [comment]
I think @doggy8088 is correct, and unfortunately Chrome itself isn't letting this extension run on `chrome.google.com`. I haven't been able to find any documentation that explains this, though, so any pointers would be welcome.

**Comment by toughdiamond** [comment]
I just tried it again (still using Firefox 68.8esr), and it seems to be working now :-)  Strangely, the first outgoing Facebook link I clicked this time retained the ?fbclid string, but subsequent ones were all fine.  So I went back and tried the first one again and that was clean too.  Best guess, something hadn't kicked in immediately, and I didn't persevere the first time.  It could be that I was reluctant before to keep clicking on links while it didn't seem to be cleaning them, as I would then have got tracked every time, but this time round I realised that all I have to do is turn the wireless off to stop that happening - the external pages don't open of course but I can still inspect the URLs that way. 
       Anyway I'll keep testing, and report back if anything goes wrong.
